swagger-templates/src/main/resources/OktaJava/modelInnerEnum.mustache

```diff
@@ -13,36 +13,36 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 }}
-  /**
-   * {{^description}}Enum {{{name}}}{{/description}}{{#description}}{{{description}}}{{/description}}
-   */
-  public enum {{#datatypeWithEnum}}{{{.}}}{{/datatypeWithEnum}}{{^datatypeWithEnum}}{{classname}}{{/datatypeWithEnum}} {
-    {{#gson}}
-        {{#allowableValues}}
-            {{#enumVars}}
-    @SerializedName({{#isInteger}}"{{/isInteger}}{{#isDouble}}"{{/isDouble}}{{#isLong}}"{{/isLong}}{{#isFloat}}"{{/isFloat}}{{{value}}}{{#isInteger}}"{{/isInteger}}{{#isDouble}}"{{/isDouble}}{{#isLong}}"{{/isLong}}{{#isFloat}}"{{/isFloat}})
-    {{{name}}}({{{value}}}){{^-last}},
-    {{/-last}}{{#-last}};{{/-last}}
-            {{/enumVars}}
-        {{/allowableValues}}
-    {{/gson}}
-    {{^gson}}
-        {{#allowableValues}}
-            {{#enumVars}}
-    {{{name}}}({{{value}}}){{^-last}},
-    {{/-last}}{{#-last}};{{/-last}}
-            {{/enumVars}}
-        {{/allowableValues}}
-    {{/gson}}
+    /**
```

Comment:
No changes except for white space. Restore this file so it doesn't show up in the diff.

Reply:
This file is not well-formatted. Code generated by this template shifted and looks not pretty.